src/test/java/org/matsim/analysis/RunOfflineAirPollutionAnalysisTest.java

```diff
@@ -32,13 +32,13 @@
 	@Test
```

Comment:
You can just add an @ignore annotation and the test will be ignored - instead of commenting out the functionality